app/broadcast_areas/__init__.py

```diff
@@ -65,7 +68,15 @@ def __init__(self, row):
         self.items = BroadcastAreasRepository().get_all_areas_for_library(self.id)
 
     def get_examples(self):
-        return BroadcastAreasRepository().get_library_description(self.id)
+        # we show up to four things. three areas, then either a fourth area if there are exactly four, or "and X more".
+        areas_to_show = [area[1] for area in sorted(self.items, key=itemgetter(1))[:4]]
```

Comment:
You don’t need to look directly at `items` here because `self` is a `SerialisedModelCollection`:
```suggestion
        areas_to_show = [area.name for area in self[:4]]
```